Readme.md

```diff
@@ -95,6 +95,20 @@ more details in this DZone article about performance comparison.
 built with the support of `htmlflow.DynamicHtml`.
 Under the unit tests package [htmlflow.test.views](src/test/java/htmlflow/test/views) you can also find more complex views with dynamic partial blocks or check some examples of [petclinic views](https://github.com/xmlet/spring-petclinic/tree/master/src/main/java/org/springframework/samples/petclinic/views).
 
+## Why another templating enginge
+
+Every popular general purpose language has its own template engine. Java being popular for quite some time has several of them.
```

Comment:
Rephrase. Maybe: "Java has been well-known for providing several of them, for quite some time."